lib/plurimath/math/symbol.rb

```diff
@@ -19,11 +19,20 @@ def to_asciimath
       end
 
       def to_mathml_without_math_tag
-        "<mo>#{mathml_symbol_value}</mo>"
+        mathml_symbol_value
       end
 
       def mathml_symbol_value
-        Mathml::Constants::UNICODE_SYMBOLS.invert[value] || value
+        unicode_symbols = Mathml::Constants::UNICODE_SYMBOLS.invert
```

Comment:
Can you extract this into a separate method somewhere? Unicode conversions are probably common.